dev-tools/kubernetes/metricbeat/manifest.run.dev.yaml

```diff
@@ -163,8 +163,7 @@ spec:
         imagePullPolicy: Never
         args: [
           "-c", "/etc/metricbeat.yml",
-          "-e",
-          "-system.hostfs=/hostfs",
```

Comment:
Why this change?

Reply:
I think I saw a warning about this parameter being deprecated. I'll test again and add a comment here

Reply:
Nevermind I have reversed this change.

Reply:
If there is a need to change it let's do it in a separate PR.